sycl/doc/extensions/InvokeSIMD/InvokeSIMD.asciidoc

```diff
@@ -21,27 +21,26 @@ IMPORTANT: This specification is a draft.
 
 NOTE: Khronos(R) is a registered trademark and SYCL(TM) and SPIR(TM) are trademarks of The Khronos Group Inc.  OpenCL(TM) is a trademark of Apple Inc. used by permission by Khronos.
 
-NOTE: This document is better viewed when rendered as html with asciidoctor.  GitHub does not render image icons.
-
 This extension introduces a mechanism to invoke explicitly vectorized functions
 written using SIMD types from a SYCL kernel.
 
 == Notice
 
-Copyright (c) 2021 Intel Corporation.  All rights reserved.
+Copyright (c) 2021-2022 Intel Corporation.  All rights reserved.
 
 == Status
 
 Working Draft
 
-This is a preview extension specification, intended to provide early access to a feature for review and community feedback. When the feature matures, this specification may be released as a formal extension.
-
-Because the interfaces defined by this specification are not final and are subject to change they are not intended to be used by shipping software products.
+This is a proposed extension specification, intended to gather community
```

Comment:
This is just a formatting comment, but does asciidoc care about newlines in paragraphs? I ask because it seems like the  previous version had long lines, while this change is re-flowed.

Reply:
asciidoc doesn't, but other reviewers have complained about long lines in extension documentation.  Some of the boilerplate content from older specifications still have long lines; I saw this as an opportunity to update InvokeSIMD to use some of the newer boilerplate content (with short lines).